index.js

```diff
@@ -10,17 +10,20 @@ const production = process.env.NODE_ENV === 'production';
 
 if (production) Sentry.init({ dsn: process.env.SENTRY_URL, environment: process.env.NODE_ENV });
 
-const errorHandler = () => ({
+const errorHandler = (blockList = []) => ({
```

Comment:
utilizar um objeto, para que fique explicito o que o parâmetro faz
```suggestion
const errorHandler = ({ blockEvents = [] } = {}) => ({
```

Reply:
não entendi como isso deixaria mais explícito.

Reply:
Não é o nome da variável, mas sim a chamada.
O exemplo que você utiliza é o seguinte:

```javascript
errorHandler([401, 403, 404]).logError(origin, err500);
```

ficaria assim
```javascript
errorHandler({ blockEvents: [401, 403, 404] }).logError(origin, err500);
```

Reply:
https://github.com/rankmyapp/error-handler/pull/2/commits/104add7c07a06641d548c4b5b12b55b7c65c80d3